pkg/server/container_start.go

```diff
@@ -103,7 +103,7 @@ func (c *criContainerdService) startContainer(ctx context.Context, id string, me
 	sandboxConfig := sandboxMeta.Config
 	sandboxID := meta.SandboxID
 	// Make sure sandbox is running.
-	sandboxInfo, err := c.containerService.Info(ctx, &execution.InfoRequest{ContainerID: sandboxID})
+	sandboxInfo, err := c.taskService.Info(ctx, &execution.InfoRequest{ContainerID: sandboxID})
```

Comment:
Yeah wasn't sure how far we wanted to push the use of the term "task" as a replacement for "container" given that task isn't in CRI/Kube and really is just a new naming convention for docker/containerd.